src/android/GooglePlus.java

```diff
@@ -194,7 +212,7 @@ private synchronized void buildGoogleApiClient(JSONObject clientOptions) throws
      */
     private void signIn() {
         Intent signInIntent = Auth.GoogleSignInApi.getSignInIntent(this.mGoogleApiClient);
-        cordova.getActivity().startActivityForResult(signInIntent, RC_GOOGLEPLUS);
+        this.signInActivityLauncher.launch(signInIntent);
```

Comment:
https://medium.com/@patelsneh18/startactvivityforresult-deprecated-alternative-and-using-it-outside-activity-class-bc9331cf896